src/rgw/CMakeLists.txt

```diff
@@ -269,32 +208,31 @@ endif()
 
 set(rgw_a_srcs
   rgw_auth_keystone.cc
+  rgw_bucket_sync.cc
   rgw_client_io.cc
   rgw_frontend.cc
   rgw_http_client_curl.cc
   rgw_loadgen.cc
   rgw_log.cc
   rgw_lua_request.cc
+  rgw_opa.cc
+  rgw_os_lib.cc
   rgw_period_pusher.cc
+  rgw_process.cc
   rgw_realm_reloader.cc
   rgw_realm_watcher.cc
```

Comment:
Hmm... I had realm_watcher below, but maybe it goes with real_reloader?  It's only being created for RadosStore currently, but it's not conditionally compiled.